packages/framework/collabspaces/src/collabSpaces.ts

```diff
@@ -761,7 +764,11 @@ export class CollabSpacesRuntime
 
 		let row;
 		for (row = 1; row < rowCount; row++) {
-			if (this.areEqualUuid(this.matrix.getCell(row, 0) as unknown as uuidType, rowId)) {
+			const uuidTypeVal = this.matrix.getCell(row, 0) as unknown as uuidType;
+			if (uuidTypeVal === undefined) {
+				throw new Error(`uuidTypeVal is undefined for row ${row}`);
+			}
```

Comment:
Maybe move this check into areEqualUuid()? You would lose "row" vs. "column" text verbatim - not sure if it matters.

Reply:
What I want here is, independently of the row we want to find, is to find inconsistent cells on the matrix row/column. Until we get to the row we want,  I want to capture regressions as soon as possible.
 Does it help ?